### DISCUSSION TITLE
Documentation fix suggestion

### DISCUSSION THREAD

**Discussion by dehidehidehi** [saleweaver/python-amazon-sp-api#102] [Feature requests]
Just a small comment, I ended up a little frustrated at the docs:
[https://python-amazon-sp-api.readthedocs.io/en/latest/endpoints/feeds/#sp_api.api.Feeds.get_feed_result_document](https://python-amazon-sp-api.readthedocs.io/en/latest/endpoints/feeds/#sp_api.api.Feeds.get_feed_result_document)

The parameter currently is : 
Parameters
feed_id – str

But should actually be:
result_feed_document_id - str

I believe we should be as explicit as possible, especially in this case when Amazon's API returns lots of ids such as "feed_id", "result_feed_document_id", "feed_document_id".

**Comment by saleweaver** [comment]
Thanks, will change in the next release.
